### PR TITLE
[minor] perl-critic over LaTeXML.pm

### PR DESCRIPTION
**Pull request by dginev**
Unsure if this PR is worth the trouble...

I got a scary `severity: 5` error from perl critic when i tried to change LaTeXML.pm and decided to at least fix that. The downside is that it requires a new dependency `IO::Interactive`, and lead to reformatting a couple more files.

Upside is that Critic is happier. Should be functionally equivalent in all usual cases I can think of.

### CODE REVIEW

**Comment by dginev** [comment]
@brucemiller I backed out of the unnecessary (albeit critic-suggested) new dependency, the rest of the changes are quite minor. Maybe mergeable in this shape?

File: Makefile.PL [hunk 1]
```diff
@@ -32,7 +32,7 @@ our $TEXMF;
 my ($texmfspec) = grep { /^TEXMF/ } @ARGV;
 if ($texmfspec && $texmfspec =~ /^TEXMF\s*=(.*)$/) {
   $TEXMF = $1;
-  @ARGV = grep { $_ ne $texmfspec } @ARGV;    # Remove so MakeMaker doesn't fret.
+  local @ARGV = grep { $_ ne $texmfspec } @ARGV;    # Remove so MakeMaker doesn't fret.
```
Comment (brucemiller) [inline Makefile.PL]: doesn't local here restrict the effect to the body of the if? If so, it's pretty much a NOOP (& doesn't have desired effect)

Comment (dginev) [reply Makefile.PL]: Indeed, this was a harmful change, apologies. It bugs me that removing it once again makes any change to Makefile.PL requires `--no-verify` on commit, but I shouldn't have broken it. Critic's insisting messages pushed me too far.

**Comment by brucemiller** [comment]
I guess this all looks right now. Lotsa gratuitous spacing changes, though.  But Thanks!

**Comment by dginev** [comment]
Please redirect all whitespace gratitude to perltidy, I take no responsibility for its decisions :>

**Pull request merged**